lightning/src/ln/channelmanager.rs

```diff
@@ -5703,7 +5735,8 @@ where
 						_ => pending_forward_info
 					}
 				};
-				try_chan_entry!(self, chan.get_mut().update_add_htlc(&msg, pending_forward_info, create_pending_htlc_status, &self.fee_estimator, &self.logger), chan);
+				let logger= WithContext { logger: &self.logger, peer_id: Some(chan.get().context.get_counterparty_node_id()), channel_id: Some(chan.get().context.channel_id()) };
```

Comment:
nit: space before `=` here and elsewhere.

Reply:
will fix